IRC.md

```diff
@@ -1,111 +1,238 @@
 - Your nickname must be your Twitch nickname in lowercase
 - Your password should be an OAuth token with the `chat_login` scope. The token must have the prefix of `oauth:`. For example, if you have the token `abcd`, you send `oauth:abcd`. You can get a token for your account with this helpful [page](http://twitchapps.com/tmi/) (thanks to [Andrew Bashore](https://github.com/bashtech)!).
 
 ## Upon a Successful Connection
+
 A successful connection session will look something like this:
+
 ```
 < PASS oauth:twitch_oauth_token
 < NICK twitch_username
-> :tmi.twitch.tv 001 twitch_username :connected to TMI
-> :tmi.twitch.tv 002 twitch_username :your host is TMI
-> :tmi.twitch.tv 003 twitch_username :this server is pretty new
-> :tmi.twitch.tv 004 twitch_username tmi.twitch.tv 0.0.1 w n
-> :tmi.twitch.tv 375 twitch_username :- tmi.twitch.tv Message of the day - 
-> :tmi.twitch.tv 372 twitch_username :- not much to say here
-> :tmi.twitch.tv 376 twitch_username :End of /MOTD command
+> :tmi.twitch.tv 001 twitch_username :Welcome, GLHF!
+> :tmi.twitch.tv 002 twitch_username :Your host is tmi.twitch.tv
+> :tmi.twitch.tv 003 twitch_username :This server is rather new
+> :tmi.twitch.tv 004 twitch_username :-
+> :tmi.twitch.tv 375 twitch_username :-
+> :tmi.twitch.tv 372 twitch_username :You are in a maze of twisty passages, all alike.
+> :tmi.twitch.tv 376 twitch_username :>
 ```
 
 About once every five minutes, you will receive a `PING tmi.twitch.tv` from the server, in order to ensure that your connection to the server is not prematurely terminated, you should reply with `PONG tmi.twitch.tv`.
 
 ## On an Unsuccessful Connection
+
 If your connection fails for any reason, you will be disconnected from the server. Some common reasons for being disconnected immediately include:
 
 - Connecting on the wrong port
 - Connecting to the wrong server
 - Using an incorrect username and/or password
- 
+
 ## Command & Message Limit
-- If you send more than 20 commands or messages to the server within a 30 second period, you will get locked out for 8 hours automatically. These are **not** lifted so please be careful when working with IRC!
+
+- If you send more than 20 commands or messages to the server within a 30 second period, you will be locked out for 8 hours automatically. These are **not** lifted so please be careful when working with IRC!
+- The above limit is elevated to 100/30 for users who *only* send messages/commands to channels in which they have Moderator/Operator status.
 
 ## Commands you can send
+
 If you send an invalid command, you will receive a 421 numeric back:
+
 ```
-< CAP REQ :multi-prefix
-> :tmi.twitch.tv 421 you CAP :Unknown command
+< WHO #channel
+> :tmi.twitch.tv 421 twitch_username WHO :Unknown command
 ```
 
 A brief list of commands supported by our IRC server include:
-### JOIN: Opening up a chat room (and obtaining a list of users)
-**JOIN** *#channelname*
-
-Notes:
+#### JOIN: Opening up a chat room
```

Comment:
@Fugiman is there a particular reason? They are still distinct from smaller text with this sizing, and are much more distinct from the larger header styles. This change makes the document easier to read.